### PR TITLE
Improve case import UX

### PR DESCRIPTION
**Pull request by millerdev**
Addresses a probable cause of https://dimagi.atlassian.net/browse/SAAS-14825: importing system properties results in duplicate properties in OData feeds.

Behavioral changes:
- System case properties are no longer selected for import by default.
  image
- (minor) Select "caseid" as default search column if present in the worksheet. The previous default of the first column in the worksheet ("number" if using an export sheet as template) was undesirable.
  image

Other minor visual UX changes:
- Hide Excel Field **->** Case Property arrow when field will not be imported (see screenshot above).

- Case property suggestions are now presented as info messages rather than warnings.
  **Before**
  image
  **After**
  image

- Warn when importing to a system property since that can result in undesirable outcomes such as duplicate columns in case exports, some of which will be ineffective because system and user property namespacing is not handled consistently.
  image

:blowfish: Review by commit.

The original discovery and the first commit were done as a group with @jingcheng16 and @mjriley 

### Safety story

This changes the default behavior of the case importer. Most of the changes are minor visual tweaks.

### Automated test coverage

No. Tested in local dev environment.

### QA Plan

Does this need QA?

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

**Review by snopoke — APPROVED** [review]
Great set of changes!

**Review by mjriley — APPROVED** [review]
LGTM

File: corehq/apps/case_importer/static/case_importer/js/excel_fields.js [hunk 1]
```diff
@@ -102,6 +100,9 @@ hqDefine('case_importer/js/excel_fields', [
                 });
                 return _.chain(suggestions).sortBy('distance').pluck('field').value();
             });
+            row.isSystemProperty = ko.computed(function () {
+                return row.isCustom() && systemFields.contains(row.customCaseField());
+            });
```
Comment (mjriley) [inline corehq/apps/case_importer/static/case_importer/js/excel_fields.js:105]: With custom's definition essentially being changed to 'not system property', should we rewrite `isSystemProperty` to not reference whether it is custom or not? Seems like a system property is any property whose name is included in the system fields

**Review by mjriley — APPROVED** [review]
Left one non-blocking comment, but otherwise looks good

File: corehq/apps/case_importer/static/case_importer/js/excel_fields.js (hunk 1 above)
Comment (millerdev) [reply corehq/apps/case_importer/static/case_importer/js/excel_fields.js:105]: I don't think so. We only want to return true here if the user has checked the "Create new property" checkbox (which sets `isCustom` to true) and the custom field name matches a system property.

If a field is not custom then the property is either not set (will not be imported) or is one of the `show_in_menu` properties, which I believe do not include system properties.

**Pull request merged**